subworkflows/local/consensus/main.nf

```diff
@@ -92,16 +92,20 @@ workflow CONSENSUS {
         FGBIO_ZIPPERBAMS(ch_zipperbam)
 
         ch_versions = ch_versions.mix(FGBIO_ZIPPERBAMS.out.versions)
-    
-    
+
+
     // 1.3: Mapped BAM => Grouped BAM
 
-        // 1.3: Mapped BAM => Grouped BAM
-        def ch_strategy = Channel.value( (params.umi_group_strategy ?: 'Adjacency') as String )
+        def valid_strategies = ['identity', 'edit', 'adjacency', 'paired']
+        def umi_strategy = (params.umi_group_strategy ?: 'adjacency').toLowerCase()
+        if ( !valid_strategies.contains(umi_strategy) ) {
+            exit 1, "Invalid value for --umi_group_strategy: '${params.umi_group_strategy}'. Allowed values: ${valid_strategies.join(', ')}"
+        }
+        def ch_strategy = Channel.value(umi_strategy)
```

Comment:
```suggestion
        def ch_strategy = Channel.value(params.umi_strategy)
```

You don't need to check this here. You can add allowed options to the parameter in the schema. (Use `nf-core schema build`, go to that parameter and set the allowed options for that parameter). This way the validation will happens before the pipeline actually starts